[debug] not up to date. inChanged = true, force = false
[debug] Updating ...
[warn] 
[warn] 	Note: Unresolved dependencies path:
[error] sbt.librarymanagement.ResolveException: Error downloading org.scalafx:scalafx_3:17.0.0-R24
[error]   Not found
[error]   Not found
[error]   not found: C:\Users\fsome\.ivy2\localorg.scalafx\scalafx_3\17.0.0-R24\ivys\ivy.xml
[error]   not found: https://repo1.maven.org/maven2/org/scalafx/scalafx_3/17.0.0-R24/scalafx_3-17.0.0-R24.pom
[error] Error downloading org.scalafx:scalafx-swing_3:17.0.0-R24
[error]   Not found
[error]   Not found
[error]   not found: C:\Users\fsome\.ivy2\localorg.scalafx\scalafx-swing_3\17.0.0-R24\ivys\ivy.xml
[error]   not found: https://repo1.maven.org/maven2/org/scalafx/scalafx-swing_3/17.0.0-R24/scalafx-swing_3-17.0.0-R24.pom
[error] 	at lmcoursier.CoursierDependencyResolution.unresolvedWarningOrThrow(CoursierDependencyResolution.scala:346)
[error] 	at lmcoursier.CoursierDependencyResolution.$anonfun$update$39(CoursierDependencyResolution.scala:315)
[error] 	at scala.util.Either$LeftProjection.map(Either.scala:573)
[error] 	at lmcoursier.CoursierDependencyResolution.update(CoursierDependencyResolution.scala:315)
[error] 	at sbt.librarymanagement.DependencyResolution.update(DependencyResolution.scala:60)
[error] 	at sbt.internal.LibraryManagement$.resolve$1(LibraryManagement.scala:60)
[error] 	at sbt.internal.LibraryManagement$.$anonfun$cachedUpdate$12(LibraryManagement.scala:142)
[error] 	at sbt.util.Tracked$.$anonfun$lastOutput$1(Tracked.scala:74)
[error] 	at sbt.internal.LibraryManagement$.$anonfun$cachedUpdate$11(LibraryManagement.scala:144)
[error] 	at sbt.internal.LibraryManagement$.$anonfun$cachedUpdate$11$adapted(LibraryManagement.scala:131)
[error] 	at sbt.util.Tracked$.$anonfun$inputChangedW$1(Tracked.scala:220)
[error] 	at sbt.internal.LibraryManagement$.cachedUpdate(LibraryManagement.scala:169)
[error] 	at sbt.Classpaths$.$anonfun$updateTask0$1(Defaults.scala:3888)
[error] 	at scala.Function1.$anonfun$compose$1(Function1.scala:49)
[error] 	at sbt.internal.util.$tilde$greater.$anonfun$$u2219$1(TypeFunctions.scala:63)
[error] 	at sbt.std.Transform$$anon$4.work(Transform.scala:69)
[error] 	at sbt.Execute.$anonfun$submit$2(Execute.scala:283)
[error] 	at sbt.internal.util.ErrorHandling$.wideConvert(ErrorHandling.scala:24)
[error] 	at sbt.Execute.work(Execute.scala:292)
[error] 	at sbt.Execute.$anonfun$submit$1(Execute.scala:283)
[error] 	at sbt.ConcurrentRestrictions$$anon$4.$anonfun$submitValid$1(ConcurrentRestrictions.scala:265)
[error] 	at sbt.CompletionService$$anon$2.call(CompletionService.scala:65)
[error] 	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
[error] 	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
[error] 	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
[error] 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
[error] 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
[error] 	at java.base/java.lang.Thread.run(Thread.java:833)
[error] (update) sbt.librarymanagement.ResolveException: Error downloading org.scalafx:scalafx_3:17.0.0-R24
[error]   Not found
[error]   Not found
[error]   not found: C:\Users\fsome\.ivy2\localorg.scalafx\scalafx_3\17.0.0-R24\ivys\ivy.xml
[error]   not found: https://repo1.maven.org/maven2/org/scalafx/scalafx_3/17.0.0-R24/scalafx_3-17.0.0-R24.pom
[error] Error downloading org.scalafx:scalafx-swing_3:17.0.0-R24
[error]   Not found
[error]   Not found
[error]   not found: C:\Users\fsome\.ivy2\localorg.scalafx\scalafx-swing_3\17.0.0-R24\ivys\ivy.xml
[error]   not found: https://repo1.maven.org/maven2/org/scalafx/scalafx-swing_3/17.0.0-R24/scalafx-swing_3-17.0.0-R24.pom
